gke-deploy/cloudbuild.yaml

```diff
@@ -6,10 +6,10 @@ steps:
 
 # Build the gke-deploy binary and put into the builder image.
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/gke-deploy', '.']
-- name: 'gcr.io/$PROJECT_ID/gke-deploy'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/gke-deploy:keychain', '.']
+- name: 'gcr.io/$PROJECT_ID/gke-deploy:keychain'
   args: ['--help']
 
 images:
-- 'gcr.io/$PROJECT_ID/gke-deploy'
+- 'gcr.io/$PROJECT_ID/gke-deploy:keychain'
```

Comment:
Why is `:keychain` added now? This will change how we release the image.

Reply:
Sorry, this was accidentally included

Reply:
Thanks for catching this